Grbl_Esp32/Custom/CoreXY.cpp

```diff
@@ -314,7 +314,7 @@ void forward_kinematics(float* position) {
     // apply the forward kinemetics to the machine coordinates
     // https://corexy.com/theory.html
     //calc_fwd[X_AXIS] = 0.5 / geometry_factor * (position[X_AXIS] + position[Y_AXIS]);
-    calc_fwd[X_AXIS] = ((0.5 * (print_position[X_AXIS] + print_position[Y_AXIS]) * geometry_factor) - wco[X_AXIS]);
+    calc_fwd[X_AXIS] = ((0.5 * (print_position[X_AXIS] + print_position[Y_AXIS]) / geometry_factor) - wco[X_AXIS]);
```

Comment:
To my way of thinking, it would be clearer to have COREXY_X_GAIN and COREXY_Y_GAIN defines, instead of the hardcoded 0.5 further modified by geometry_factor which is then selected by ifdef MIDTBOT.

I also find it confusing that geometry_factor applies to both X and Y in inverse kinematics, but only to X in forward kinematics.  Is this a bug or a subtle point that I am missing?